view/frontend/templates/info/laybuy.phtml

```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * @var $block \Laybuy\Laybuy\Block\Info
+ */
+?>
+<?php echo $block->escapeHtml($block->getMethod()->getTitle())?>
```

Comment:
whatever happens too <?= ?

Reply:
Yeah that's a good point, this was copy-paste from the adminhtml version